storybook/README.md

```diff
@@ -0,0 +1,322 @@
+
+```ts
+{
+  /**
+   * Define the properties to pass for this specific story. Often,
+   * `Default` stories wouldn't even need this.
+   */
+  args: Partial< MyComponentProps >;
+
+  /**
+   * You will rarely need this, as story names are automatically generated
+   * from your constant name. But that's good to know if you ever need.
+   */
+  storyName: string;
+}
+```
+
+Full official docs: https://storybook.js.org/docs/react/writing-stories/introduction
+
+### Snippets
+
+If you are using [VSCode](https://code.visualstudio.com/), this repo includes some helpful snippets that will become available inside `.ts` and `.tsx` files. You can find them inside `.vscode/storybook.code-snippets`.
+
+The `sbs` (“Storybook story”) will scaffold the entire code in the [section above](#scaffold-tour). If you have respected the naming conventions, it will also property import your component and the properties with the correct names, saving you a bunch of time.
+
+The `sbt` (“Storybook story template”) will create a new story by binding your default template and prompting you to provide specific arguments.
+
+## FAQ
+
+### What should constitute a story?
+
+Stories should show the component in its different shapes.
+
+The most obvious way is when a component has a few variations (such as a button with a `primary` attribute being larger and bolder, for example).
+
+However, a common misconception is that, since full controls are enabled and allow the user to explore the component by playing around with its properties interactively, there is no need to provide stories which are just the component with different default properties.
+
+While this is true for many cases (often you don't need to create a different story by replacing the `label` of a component, if that's just text; though it might be argued that it could be interesting to have a story showing how a very long text behaves, especially in [combined stories](#can-i-create-stories-with-mixed-components)), it is very useful to create stories for components in their loading or error states.
+
+### Can I create stories with mixed components?
+
+Yes, and it'd be awesome to see how our components interact with each other, especially in the context of blocks. At the time of writing, we have no such stories, however, here is a link to the official docs if you want to give a go at implementing this: https://storybook.js.org/docs/react/workflows/stories-for-multiple-components
+
+## Tips
+
+### One file per component
+
+While stories for closely related components could technically be living within the same file, we advise that you keep one file per component, as this works best with the automatic organization of the components within storybook and using [`default` exports in the Story](https://storybook.js.org/docs/react/api/csf#default-export).
+
+An exception to this, might be when there is only one mainly used component and the other defined ones are just used internally.
```

Comment:
```suggestion
An exception to this might be when there is only one mainly used component and the other defined ones are just used internally.
```

Reply:
I feel this comma sounds a bit awkward in the middle of the sentence. Perhaps, if the sentence would have been `As an exception to this, […there is that]`, I would have seen it as more appropriate… 🤔